cephmetrics.spec.in

```diff
@@ -33,6 +33,7 @@ The monitoring service with web frontend for Ceph storage clusters providing sev
 %package grafana-plugins
 Summary:	Vonage plugin for graphana
 Requires:	grafana
+Requires:	PyYAML
```

Comment:
If it is needed by dashUpdater.py, we should make it a dependency of the main cephmetrics package which contains it (i.e. we should put it somewhere around the line 25 of the spec file).

I grepped the sources and the only `import yaml` clause that I found was in the dashUpdater.py.